packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -833,6 +834,27 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                <tr>
+                  <th>{_('vmBootFirmware')}</th>
+                  <td>
+                    <select
+                      className='form-control'
+                      onChange={event => {
+                        editVm(vm, {
+                          hvmBootFirmware: getEventValue(event),
+                        })
+                        info(_('vmBootFirmwareWarning'))
```

Comment:
Ask for confirmation instead of displaying a notice.

And only in the when the selected value is appropriate.

Reply:
Do you mean that we should ask a confirmation only when `UEFI` is selected ?

Reply:
If `uefi` is the risky setting, then yes we should ask a confirmation for it.

/cc @olivierlambert What do you think?

Reply:
Risky, well I don't think so, but it's experimental. I'm not against a confirmation modal.